frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -145,14 +155,25 @@ const ManagePolicyPage = ({
       ? parseInt(queryParams?.inherited_page, 10)
       : 0)();
 
-  const page = initialPage;
+  // const page = initialPage;
   const showInheritedTable = initialShowInheritedTable;
-  const inheritedPage = initialInheritedPage;
-  const searchQuery = initialSearchQuery;
+  // const inheritedPage = initialInheritedPage;
+  // const searchQuery = initialSearchQuery;
```

Comment:
Did you mean to leave these comments?

Reply:
not at all thanks, was removing commented out code and console logs for help referencing once backend was merged